### DISCUSSION TITLE
Docker Issues

### DISCUSSION THREAD

**Discussion by jasonpaulso** [AllYourBot/hostedgpt#386] [General]
I am having an issue that I can't resolve where trying to bring up the project the 'base' container continuously restarts with "... Running db:prepare... Done ensuring encryption is setup...". Have you seen this?

**Comment by keithschacht** [comment]
@jasonpaulso I haven't experienced that before. That's odd! And I can't think of anything that has changed recently about Docker. I'll review the git history just to see if I'm forgetting something.

But the few times I've had an issue with Docker what I do is. Can you try these steps?

open docker desktop
delete all containers, images, volumes, and builds associated with the app
do `git pull` to make sure you've got the latest
Then re-run `docker compose up --build`

**Comment by keithschacht** [comment]
@jasonpaulso actually, I forgot about this discussion. Is this your issue? See the last comment: https://github.com/AllYourBot/hostedgpt/issues/359

**Comment by keithschacht** [comment]
@jasonpaulso I just repo'd the issue for myself. :( I'm working to fix it now.

**Comment by keithschacht** [comment]
@jasonpaulso I pushed my last fix and did a fresh setup, start to finish. I believe it's all working now. Sorry about the trouble with this! I'm actually glad that you flagged it though because I had inadvertently broken Docker for everyone and didn't realize it. I need to switch back and forth between the two setups more frequently.

**Reply by jasonpaulso** [reply]
Just checked myself and it looks like we're back in business! I'll have time to pick up where I left off beginning tomorrow.
